ruby/exercises/numeros_romanos.rb

```diff
@@ -0,0 +1,76 @@
+class Conversiones
+  @@respuesta = ""
+
+  def romanos(data)
+    return 'Numero invalido' unless data >= 1 && data <= 3999
+    @@respuesta = ""
+    @@respuesta = genera_busquedas(data)  
+  end
+
+  def numeros(data)
+    romanos = {'I': 1, 'V': 5, 'X': 10, 'L': 50, 'C': 100, 'D': 500, 'M': 1000}
```

Comment:
Esto podria ser una constante. las constantes son definidas en mayusculas dentro de las clases y se declaran al inicio
```suggestion
    ROMANOS = {'I': 1, 'V': 5, 'X': 10, 'L': 50, 'C': 100, 'D': 500, 'M': 1000}
```